### PR TITLE
Makefile and Dockerfile template updates

### PR DESCRIPTION
**Pull request by phillipross**
- Updated dockerfile templates to reflect new maintainer labeling, debian package
  installation, and alpine build process
- Updated Makefile to change mdillon docker repo to postgis docker repo
- Updated update.sh script to use more modern parsing of debian packages to
  accommodate multiple debian versions in a similar way to how the postgres image
  built as part of docker-library accomplishes it

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,18 +5,18 @@ services: docker
...
   - VERSION=12-2.5 VARIANT=alpine
+  - VERSION=11-2.5
+  - VERSION=11-2.5 VARIANT=alpine
+  - VERSION=10-2.5
+  - VERSION=10-2.5 VARIANT=alpine
+  - VERSION=9.6-2.5
+  - VERSION=9.6-2.5 VARIANT=alpine
+  - VERSION=9.5-2.5
+  - VERSION=9.5-2.5 VARIANT=alpine
+  - VERSION=9.4-2.5
```
Comment (dbaston) [inline .travis.yml]: 9.4 goes EOL on Thursday; consider dropping?

Comment (phillipross) [reply .travis.yml]: good point, and now's a good time to trim down things to be maintained.  Thanks @dbaston

**Comment by smellman** [comment]
@phillipross I recommend to add travis_retry command.
https://github.com/postgis/docker-postgis/pull/164/commits/5345e730339788be4ad169f8452f9c3482a3b86f
In my case travis ci failed in using wget command in alpine build.
https://travis-ci.org/postgis/docker-postgis/jobs/646651624

**Comment by phillipross** [comment]
@smellman Thanks for the suggestion.  The way you had it configured, I think it would retry building the docker image if it fails for any reason, not just a network timeout, true?  I have a feeling that this might be too broad and such fault tolerance should probably be handled in the Dockerfile itself when it's pulling external dependencies.

**Pull request merged**